internal/logging/logger.go

```diff
@@ -44,6 +50,19 @@ func UseServerLogger() {
 	L = newLogger(os.Stderr)
 }
 
+// UseFileLogger changes L to a logger that writes log output to a file that is
+// rotated.
+func UseFileLogger(filepath string) {
```

Comment:
Moved from `file.go` since that file contained only this one function (the type was unused), and so that all things that modify `L` are close together in this one file.